src/puppetlabs/puppetdb/cli/benchmark.clj

```diff
@@ -209,34 +209,45 @@
       (when report
         (future
           (try
-            (client/submit-report base-url 5 (json/generate-string report))
+            (client/submit-report base-url 5 report)
             (log/infof "[%s] submitted report" host)
             (catch Exception e
               (log/errorf "[%s] failed to submit report: %s" host e)))))
       (when factset
         (future
           (try
-            (client/submit-facts base-url 4 (json/generate-string factset))
+            (client/submit-facts base-url 4 factset)
             (log/infof "[%s] submitted factset" host)
             (catch Exception e
               (log/errorf "[%s] failed to submit factset: %s" host e)))))
       (assoc state
-        :lastrun clock
-        :catalog catalog))))
-
-(defn update-host
+             :lastrun clock
+             :catalog catalog))))
```

Comment:
answer to the above is no, since only the catalog is being assoc'd here. I'm not sure why that's the case.